WordPress/wordcamp.org
# Tools: Update how linting runs on travis

Add phpcodesniffer & the WP rules to the composer install (installed at `public_html/wp-content/mu-plugins/vendor`), and set up a composer script to run phpcs & phpcbf. We can now use this both locally and in travis to make sure everyone's got phpcs installed and on the correct versions.

Travis has also been updated to use [jobs,](https://docs.travis-ci.com/user/job-lifecycle/) which will let us run multiple checks in parallel. Here we're able to run the JS & SCSS linters on the blocks project while the PHP Code Sniffer checks run, and we could set this up to include more checks in other projects later.

The travis file should be well documented, but let me know if I should add anything else.

**To test**

- Travis does the testing :) Verify that the checks passed.
- Read through the .travis.yml file to make sure my changes make sense.
